django/db/migrations/operations/base.py

```diff
@@ -146,6 +146,9 @@ def allow_migrate_model(self, connection_alias, model):
 
         return router.allow_migrate_model(connection_alias, model)
 
+    def reduce_related(self, operation, app_label):
+        return None
```

Comment:
```suggestion
        return [self]
```
Then we don't have to do `op.reduce_related(other, app_label) or [op]` and can do `op.reduce_related(other, app_label)` instead
We might want a doc string here also